packages/fast-development-site-react/src/components/shell/action-bar.tsx

```diff
@@ -6,13 +6,13 @@ import { IDevSiteDesignSystem } from "../design-system";
 export interface IShellActionBarProps {}
 
 export interface IShellActionBarManagedClasses {
-    shell__action_bar: string;
+    shell_action_bar: string;
 }
 
 const style: ComponentStyles<IShellActionBarManagedClasses, IDevSiteDesignSystem> = {
-    shell__action_bar: {
+    shell_action_bar: {
```

Comment:
should be `actionBar` since we are using _ to separate Blocks from Elements (see https://github.com/Microsoft/fast-dna/wiki/Coding-standards) -- @nicholasrice has just updated the wiki so we can bring all of our naming conventions inline